curator/templates/single.html

```diff
@@ -1,28 +1,121 @@
-<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:template-part {"slug":"header","theme":"curator","tagName":"header"} /-->
```

Comment:
Let's get rid of the theme slug here, so the templates and parts always resolve from the currently active theme:

```suggestion
<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```

Same for the footer part.